### PR TITLE
[test] Fix typo in ipcmagic module version

### PR DESCRIPTION
**Pull request by kraushm**
See also https://jira.cscs.ch/browse/SD-53111 / https://jira.cscs.ch/browse/UES-1737

### CODE REVIEW

File: cscs-checks/apps/jupyter/check_ipcmagic.py [hunk 1]
```diff
@@ -11,55 +11,63 @@
...
+
+    @performance_function('s')
+    def time(self):
+        return sn.extractsingle(r'IPCluster is ready\!\s+'
+                                r'\((?P<time>\d+) seconds\)',
+                                self.stdout, 'time', float)
 
     @run_before('run')
-    def prepare_run(self):
+    def set_local_launcher(self):
```
Comment (vkarak) [inline cscs-checks/apps/jupyter/check_ipcmagic.py:70]:
```suggestion
    def reset_launcher(self):
```

**Review by vkarak — CHANGES_REQUESTED** [review]
Just a minor style change.

**Review by vkarak — APPROVED** [review]

File: cscs-checks/apps/jupyter/check_ipcmagic.py [hunk 2]
```diff
@@ -24,38 +24,54 @@ class IPCMagicCheck(rfm.RunOnlyRegressionTest):
     executable_opts = ['tf-hvd-sgd-ipc-tf2.py']
     reference = {
         'daint:gpu': {
-            'slope': (2.0, -0.1, 0.1, None),
-            'offset': (0.0, -0.1, 0.1, None),
-            'retries': (0, None, None, None),
+            'slope': (2.0, -0.1, 0.1, 'N/A'),
+            'offset': (0.0, -0.1, 0.1, 'N/A'),
+            'retries': (0, None, None, 'N/A'),
```
Comment (vkarak) [inline cscs-checks/apps/jupyter/check_ipcmagic.py:29]: I would probably omit the units here altogether.

**Review by vkarak — CHANGES_REQUESTED** [review]

**Review by rsarm — APPROVED** [review]

File: cscs-checks/apps/jupyter/check_ipcmagic.py (hunk 2 above)
Comment (teojgo) [reply cscs-checks/apps/jupyter/check_ipcmagic.py:29]: They type checking seems to not allow mixing 3-tuples with 4-tuples

Comment (vkarak) [reply cscs-checks/apps/jupyter/check_ipcmagic.py:29]: ok, I see.

**Review by vkarak — APPROVED** [review]

**Review by teojgo — APPROVED** [review]

**Comment by teojgo** [comment]
@jenkins-cscs retry all

**Comment by vkarak** [comment]
@jenkins-cscs retry dom

**Pull request merged**